src/main/java/org/swisspush/redisques/util/DefaultRedisProvider.java

```diff
@@ -58,23 +58,23 @@ private Future<RedisAPI> setupRedisClient(){
 
     private Future<RedisAPI> connectToRedis() {
         RedisquesConfiguration config = configurationProvider.configuration();
-        String redisHost = config.getRedisHost();
-        int redisPort = config.getRedisPort();
         String redisAuth = config.getRedisAuth();
```

Comment:
Should we name this redisPassword in order that it matches redisUser?

Reply:
Not sure. Throughout the whole module it is currently named auth. In Redis config they also talk about password (requirepass).
I think I will go for the rename.

Reply:
OK, if we want to be backward compatible then we could also keep "auth", mark it deprecated and provide "password" additionally?